chapter_10/10-8.py

```diff
@@ -0,0 +1,38 @@
+# 10-8. Cats and Dogs
+
+from pathlib import Path
+
+# deal with "What folder am I working from?"
+import os
+cwd = os.getcwd()
+index = cwd.find("chapter")
+if index > 0:
+    cwd = cwd[:index]
+# else do nothing, assume we're in workspace root
+
+def readCatFile(catFile='cats.txt', animal="cats"):
```

Comment:
it looks like in pep8, the only reason they recommend using mixed case is for backwards compatibility, so it may be better to name this `read_cat_file`

Reply:
unless File is a special thing in this case?